compiler/src/org.graalvm.compiler.replacements.test/src/org/graalvm/compiler/replacements/test/StringIndexOfConstantTest.java

```diff
@@ -87,6 +87,8 @@ protected Result test(OptionValues options, ResolvedJavaMethod method, Object re
     @Override
     protected InstalledCode getCode(final ResolvedJavaMethod installedCodeOwner, StructuredGraph graph0, boolean ignoreForceCompile, boolean ignoreInstallAsDefault, OptionValues options) {
         // Force recompile if constant binding should be done
-        return super.getCode(installedCodeOwner, graph0, /* forceCompile */true, /* installAsDefault */false, options);
+        return super.getCode(installedCodeOwner, graph0, /* forceCompile */true, /*
```

Comment:
Same remark about the formater

Reply:
Hmm, yes, this is weird. I t might be my eclipse version (4.7.2) or maybe it is because I ran it with jdk11?
Anyway I'll fix this by hand.

Reply:
:+1: 
Yes different versions of eclipse produce different formatting... We use `4.5.2`.